README.rst

```diff
@@ -0,0 +1,85 @@
+LTI Consumer XBlock |Build Status|
```

Comment:
Can you add in the Travis/Coveralls status indicators up at the top. You can look over at edx-proctoring/README to see how that is done.